### DISCUSSION TITLE
LS_COLORS ignored

### DISCUSSION THREAD

**Discussion by jkroes** [marlonrichert/zsh-autocomplete#229] [Q&A] [answered]
* `zsh-autocomplete` version: 7662f28a3bca1165efeee853e2571d12837e855d
* Zsh version: ubuntu/5.8-3ubuntu1
* Framework: Oh My Zsh
* Plugin manager: none

Changing `$LS_COLORS` changes the highlighting/coloration of `ls` but not autocompleted file contents. 

Output of `ls`:

![image](https://user-images.githubusercontent.com/18478163/111569866-d1b76780-8760-11eb-8abb-4f60c51ce68c.png)

Autocomplete directory output:

![image](https://user-images.githubusercontent.com/18478163/111569931-edbb0900-8760-11eb-905e-bfcdaafac382.png)

**Comment by marlonrichert** [comment, marked as answer]
That's not handled by `zsh-autocomplete`. I made a separate plugin for that, called [Z Colors](https://github.com/marlonrichert/zcolors).
